README.md

```diff
@@ -1,9 +1,24 @@
 # sftp_jail
 
-[![Build Status](https://travis-ci.org/voxpupuli/puppet-sftp_jail.png?branch=master)](https://travis-ci.org/voxpupuli/puppet-sftp_jail)
+[![CI](https://github.com/voxpupuli/puppet-sftp_jail/actions/workflows/ci.yml/badge.svg)](https://github.com/voxpupuli/puppet-sftp_jail/actions/workflows/ci.yml)
+[![Code Coverage](https://coveralls.io/repos/github/voxpupuli/puppet-sftp_jail/badge.svg?branch=master)](https://coveralls.io/github/voxpupuli/puppet-sftp_jail)
+[![Puppet Forge](https://img.shields.io/puppetforge/v/puppet/sftp_jail.svg)](https://forge.puppetlabs.com/puppet/sftp_jail)
+[![Puppet Forge - downloads](https://img.shields.io/puppetforge/dt/puppet/sftp_jail.svg)](https://forge.puppetlabs.com/puppet/sftp_jail)
+[![Puppet Forge - endorsement](https://img.shields.io/puppetforge/e/puppet/sftp_jail.svg)](https://forge.puppetlabs.com/puppet/sftp_jail)
+[![Puppet Forge - scores](https://img.shields.io/puppetforge/f/puppet/sftp_jail.svg)](https://forge.puppetlabs.com/puppet/sftp_jail)
```

Comment:
can you please also add the badge for the LICENSE?

Reply:
okay, forgot this, will do

Reply:
done